db/fixtures.sql

```diff
@@ -31,6 +31,3 @@ insert into acos(uuid, cms_id, name, client_id)
             ('8D80925A-027E-43DD-8AED-9A501CC4CD91', 'A9992', 'ACO Large', '8D80925A-027E-43DD-8AED-9A501CC4CD91'),
             ('55954dba-d4d9-438d-bd03-453da4993fe9', 'A9993', 'ACO Extra Large', '55954dba-d4d9-438d-bd03-453da4993fe9');
 
-insert into tokens values ('d63205a8-d923-456b-a01b-0992fcb40968', 'fake.token.value', 'true');
```

Comment:
Can we also update the snapshot we're using for the unit tests to also remove the tokens entry?

```
psql -p 15432 -h 127.0.0.1 -U postgres -W bcda_test -c "DROP TABLE tokens"
make unit-test-db-snapshot
```

Reply:
Great idea @mtrang1263 .
Done in latest commit!